pkg/test/options.go

```diff
@@ -45,6 +46,7 @@ func Options(overrides ...OptionsFields) *options.Options {
 		ClusterName:             lo.FromPtrOr(opts.ClusterName, "test-cluster"),
 		ClusterEndpoint:         lo.FromPtrOr(opts.ClusterEndpoint, "https://test-cluster"),
 		IsolatedVPC:             lo.FromPtrOr(opts.IsolatedVPC, false),
+		EKSControlPlane:         lo.FromPtrOr(opts.EKSControlPlane, true),
```

Comment:
This should probably be consistent with the actual default for the project.
```suggestion
		EKSControlPlane:         lo.FromPtrOr(opts.EKSControlPlane, false),
```